pkg/stages/steps_init.go

```diff
@@ -332,7 +332,7 @@ func GetCleanupStage(sis values.System, l types.KairosLogger) []schema.Stage {
 		},
 		{
 			Name:     "Cleanup",
-			OnlyIfOs: "SLES.*|[O-o]penSUSE.*",
+			OnlyIfOs: "openSUSE.*|SUSE.*|[O-o]penSUSE.*",
```

Comment:
```suggestion
			OnlyIfOs: "openSUSE.*|SUSE.*",
```

Reply:
I swear I seen this with uppercase O somewhere, thats why I added.

But Im merging this so I will blame you if it fails :P